libraries/ui-material/Accordion/index.jsx

```diff
@@ -1,29 +1,30 @@
-// @flow
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import noop from 'lodash/noop';
 import { AccordionContainer, ChevronIcon } from '@shopgate/pwa-ui-shared';
 import { i18n } from '@shopgate/engage/core';
 import AccordionContent from './components/AccordionContent';
 import * as styles from './style';
 
-type Props = {
-  renderLabel: ({ open: boolean }) => void,
-  children: React.Node,
-  handleLabel?: string,
-  role?: string,
-  testId?: string,
-  className?: string,
-  contentClassName?: string,
-  openWithChevron?: boolean,
-  startOpened?: boolean,
-  chevronPosition?: "left"|"right",
-}
-
 /**
+ * Accordion component
  * @param {Object} props The component props.
+ * @param {Function} props.renderLabel Function that returns a React component used as header label
+ * @param {string} [props.handleLabel] Aria label for the header label
+ * @param {string} [props.role] Aria role (default `"button"`)
```

Comment:
as discussed earlier this will not create any advantage, since it seems there is no way to get an info about the default value inside the type hints for the `Accordion` component. The only way to give developers any info about the default value is to mention it in the prop description.